conans/client/configure_build_environment.py

```diff
@@ -244,8 +245,8 @@ def append(*args):
                         ret.append(arg)
             return ret
 
-        lib_paths = ['-L%s' % x for x in self.library_paths]
-        include_paths = ['-I%s' % x for x in self.include_paths]
+        lib_paths = ['-L%s' % unix_path(x) for x in self.library_paths]
```

Comment:
Sure this change in paths is enough covered by tests?

Reply:
There is no mingw autotools test, I'm adding it to the e2e issue to complete it when possible